src/game/AI/ScriptDevAI/scripts/outland/hellfire_citadel/shattered_halls/shattered_hallsScripts.cpp

```diff
@@ -270,6 +269,267 @@ struct npc_shattered_hand_centurion : public CombatAI
     }
 };
 
+/*######
+ ## npc_shattered_hand_legionnaire 16700
+ ######*/
+
+enum ShatteredHandLegionnairActions
+{
+    LEGIONNAIRE_PUMMEL,
+    LEGIONNAIRE_AURA_OF_DISCIPLIN,
+    LEGIONNAIRE_ACTION_MAX,
+    LEGIONNAIRE_CALL_FOR_REINFORCEMENTS,
+    LEGIONNAIRE_REINF_CD,
+};
+
+enum ShatteredHandLegionnair
+{
+    SPELL_AURA_OF_DISCIPLINE = 30472,
+    SPELL_PUMMEL = 15615,
+    SPELL_ENRAGE = 30485,
+
+    EMOTE_ENRAGE = 1151,
+
+    MOB_FEL_ORC = 17083,
+
+    // Guids
+    FIRST_LEGIONNAIRE_GUID = 5400150,
```

Comment:
Since its now possible, I would rewrite this as string ids. You can check if a creature has a strings id, HasStringId, and then if for some reason we have to change instance file numbering it will work without a recompile.